src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -214,7 +215,7 @@ export class NotebookMarkdownRenderer {
 			// but we might need to add _that_
 			// https://tools.ietf.org/html/rfc3986#section-3
 			if (/^[^:]+:\/*[^/]*$/.test(base)) {
-				this._baseUrls[' ' + base] = base + '/';
+				this._baseUrls[' ' + base] = path.join(base, path.sep);
 			} else {
```

Comment:
Actually this whole method is assuming that the path separator is /. So I'd revert this change and then just create an issue and we can follow up on that later (it might not even be an actual issue - not sure if these are ever going to be encoded with platform-specific path separators)